src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/core/AbstractCraftingRequestResolver.java

```diff
@@ -242,20 +245,61 @@ protected boolean createsCraftingCycle(
             return null;
         }
 
-        return createRequestsForRecipe(manager, craftableCrafting.getPrimaryOutput(), count, minCount);
+        return createRequestsForRecipe(manager, craftableCrafting, count, minCount);
     }
 
+    /**
+     * Create the crafting request entries for the overall request
+     * Will produce multiple, if the inputs don't all fit in the crafters inventory. 
```

Comment:
I think the word "input" might be what is confusing people about the doc. Better call it "ingredients" for a recipe.